<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>398</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>398</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>398</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>336</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>336</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>336</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>274</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>274</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>274</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>460</y>
      <width>162</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_del">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>150</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Result">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>460</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_plus">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>398</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_minus">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>336</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_mult">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>274</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_div">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>212</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Coma">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>460</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_L_Bracket">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>212</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="shortcut">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_R_Bracket">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>212</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="shortcut">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Exponent">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>212</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="shortcut">
     <string>^</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_str">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>35</y>
      <width>327</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>91</y>
      <width>327</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Clear">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>150</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 35, 35);
color: rgb(184, 184, 184);
padding: 5px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/icon.png&quot; width=&quot;15&quot; height=&quot;15&quot;/&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;  Сalculator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_exit">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>2</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_minimaze">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>2</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ListRemove"/>
    </property>
    <property name="iconSize">
     <size>
      <width>11</width>
      <height>15</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
